test/uiSchema_test.js

```diff
@@ -1,8 +1,10 @@
+/*eslint no-unused-vars: [2, { "varsIgnorePattern": "^d$" }]*/
+
 import { expect } from "chai";
 import React from "react";
 import { Simulate } from "react-addons-test-utils";
 
-import { createFormComponent } from "./test_utils";
+import { createFormComponent, d } from "./test_utils";
```

Comment:
`d` is a debugging helper logging the indented html version of a rendered node. We've been using this for months in the codebase.